.github/workflows/ci.yml

```
@@ -157,21 +184,169 @@
+              ls -t ~/fin-invoice-ocr-team6/backups/backup-*.tar.gz | tail -n +6 | xargs rm -f 2>/dev/null || true
+            else
+              echo "No current deployment found to backup"
+            fi
+          '
 
+      - name: Create MySQL Database Backup
+        run: |
+          ssh -o StrictHostKeyChecking=no -i ssh-key.pem ${{ secrets.USERNAME }}@${{ secrets.SERVER_IP }} '
+            # Load environment variables
+            source ~/fin-invoice-ocr-team6/shared/.env
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)